client/apps/Admin/src/pages/HourReportPage.tsx

```
@@ -0,0 +1,299 @@
+            header: 'סוג בדיווח',
+            cell: info => {
+                const reportType = info.getValue();
+                const rowId = info.row.original.id;
+                return (
+                    <div className="hour-report-page__toggle">
+                        <button
+                            className={`hour-report-page__toggle-btn ${reportType === 'summary' ? 'active' : ''}`}
+                            onClick={() => handleReportTypeToggle(rowId, 'summary')}
+                            disabled={updateReportTypeMutation.isPending}
+                        >
+                            <span className="hour-report-page__toggle-radio"></span>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)